packages/calendar/test-helpers/index.js

```diff
@@ -0,0 +1,3 @@
+export * from './CalendarObject.js';
```

Comment:
please define every export (e.g. no star exports) for better support of bundlers